topics/statistics/tutorials/intro-to-ml-with-r/tutorial.md

```diff
@@ -0,0 +1,1262 @@
+>    ```
+>
+>    ![Visualization of the Hierarchical clustering (cut at k=2) results against the first two PCs on the UCI Breast Cancer dataset](../../images/intro-to-ml-with-r/hclust-pc12-Visualization.png "Visualization of the Hierarchical clustering (cut at k=2) results against the first two PCs on the UCI Breast Cancer dataset")
+>
+{: .hands_on}
+
+
+> ### {% icon question %} Question
+>
+> 1. The hierarchical clustering performed so far, only used two methods: `euclidean` and `average`. Try experimenting with different methods. Do the final results improve?
+> 2. Obviously the cut-off selection (k=2) was not optimal. Try using different cut-offs to ensure that the final clustering could provide some context to the original question.
+>
+> > ### {% icon solution %} Solution
+> >
+> > TODO
+> >
+> {: .solution}
+{: .question}
+
+# Supervised Learning
+
+Supervised learning is the branch of Machine Learning (ML) that involves predicting labels, such as 'Survived' or 'Not'. Such models learn from labelled data, which is data that includes whether a passenger survived (called "model training"), and then predict on unlabeled data.
+
+These are generally called train and test sets because
+- You want to build a model that learns patterns in the training set, and
+- You then use the model to make predictions on the test set.
+
+We can then calculate the percentage that you got correct: this is known as the accuracy of your model.
+
+## How To Start with Supervised Learning
+
+As you might already know, a good way to approach supervised learning is the following:
+- Perform an Exploratory Data Analysis (EDA) on your data set;
+- Build a quick and dirty model, or a baseline model, which can serve as a comparison against later models that you will build;
+- Iterate this process. You will do more EDA and build another model;
+- Engineer features: take the features that you already have and combine them or extract more information from them to eventually come to the last point, which is
+- Get a model that performs better.
+
+A common practice in all supervised learning is the construction and use of the **train- and test- datasets**. This process takes all of the input randomly splits into the two datasets (training and test); the ratio of the split is usually up to the researcher, and can be anything: 80/20, 70/30, 60/40...
+
+## Supervised Learning I: classification
+
+There are various classifiers available:
+
+- **Decision Trees** – These are organized in the form of sets of questions and answers in the tree structure.
+- **Naive Bayes Classifiers** – A probabilistic machine learning model that is used for classification.
+- **K-NN Classifiers** – Based on the similarity measures like distance, it classifies new cases.
+- **Support Vector Machines** – It is a non-probabilistic binary linear classifier that builds a model to classify a case into one of the two categories.
```

Comment:
I think using the `kernel trick`, SVMs learn non-linear boundaries as well by projecting data points to higher-dimensional space. There are both linear and non-linear variants of SVMs. Thanks!